src/main/java/uk/gov/hmcts/cmc/claimstore/services/document/DocumentManagementBackedDocumentsService.java

```diff
@@ -91,8 +95,8 @@ private Claim getClaimByExternalId(String externalId, String authorisation) {
     }
 
     private DocumentDetails uploadToDocumentManagement(byte[] documentBytes,
```

Comment:
v minor - but would prob line break like - this format seems to be project: https://github.com/hmcts/cmc-claim-store/blob/65e606e964aab5c1dabc682c3fcf11aea30ea8cb/src/main/java/uk/gov/hmcts/cmc/claimstore/services/document/DocumentManagementBackedDocumentsService.java#L35